odsclient/core.py

```diff
@@ -1,13 +1,16 @@
+import json
 import os
 from ast import literal_eval
 from getpass import getpass
 
 import io
+
```

Comment:
Usually for PRs it is a good practice to avoid this kind of formatting change as they impact readability. I will accept some (see below) but not all

```suggestion

```